src/studio-home/data/api.js

```diff
@@ -16,8 +16,8 @@ export async function getStudioHomeData() {
   return camelCaseObject(data);
 }
 
-export async function getStudioHomeCourses(search) {
-  const { data } = await getAuthenticatedHttpClient().get(`${getApiBaseUrl()}/api/contentstore/v1/home/courses${search}`);
+export async function getStudioHomeCourses(search, customParams) {
+  const { data } = await getAuthenticatedHttpClient().get(`${getApiBaseUrl()}/api/contentstore/v2/home/courses${search}`, { params: customParams });
```

Comment:
Maybe we should add a comment explaining why we're using V2 of the API?

Reply:
I've added a littler explaining about it